templates/resources.html

```diff
@@ -51,6 +51,15 @@
                 </div>
                 <span class="tag is-danger">{{res.level}}</span>
                 <span class="tag is-info">{{res.type}}</span>
+                <div class="content">
+                    {% if res.reviews %}
+                        <hr />
+                        <h5 class="subtitle is-5">Reviews</h5>
+                        {% for review in res.reviews %}
+                        <p>"{{review}}"</p>
```

Comment:
Do you think italicizing it will make the review actually look like a review? Like how we see reviews on products?

Reply:
I think it is okay for it and you mention me it is above the open link it if any  changes need let me know